69.md

```diff
@@ -0,0 +1,205 @@
+
+A NIP-36 or NIP-56 report is not a suggestion to censor content, nor are they endorsements of censorship. Nostr is specifically designed to prevent any mandate to use NIP-36 or NIP-56 data. The data exists simply for contexts where it is needed or desired. Apps that support NIP-36 and NIP-56 reporting do not need to support all of the types and contexts listed below or present them to users. Likewise it's likely that many relays will instruct their moderators to ignore all the report types except those which pose serious legal issues for their organization.
+
+Some of the content types involve "in the eye of the beholder" judgement calls. Relay owners and Nostr-based communities will use and/or ignore reports only as the reports meet their particular standards. There won't be a single standard on any of these items, but rather thousands of standards.
+
+To reiterate - there is no way to censor Nostr. This isn't about censorship. It's about facilitating the content moderation necessary to be compliant with local laws and community standards that define the community of users for a particular relay or client app.
+
+
+
+Reportable Types:
+
+These types expand and standardize the "report types" defined in NIP-56. Existing NIP-56 report types (shown below as "sometype | ...") should be supported by clients for backwards compatibility, but should not otherwise be used in future development.
+
+Format: AB-cde
+ - where 'AB' is a mandatory 2 character, uppercase code of a major content type category
+ - and 'cde' is an optional 3 character, lowercase code representing a sub-category
+
+
+profanity | CL - Coarse Language / Profanity
+IH - Intolerance & Hate (does not include intolerance of intolerance)
+illegal | IL - Illegal (post/profile is illegal or advocates illegal activity)
+IL-cop - Copyright violation, piracy, intellectual property theft
+IL-csa - Child sexual abuse and/or trafficking
+IL-drg - Drug-related crime
+IL-frd - Fraud & Scams
+IL-har - Harassment / stalking / doxxing
+IL-hkr - Prostitution
+IL-idt - Identity theft / phishing
+IL-mal - Malware / viruses / ransomware
+impersonation | IM - Impersonation (consider also IL-idp)
+MI-mny - Promotion of misinformation that is likely to cause financial ruin (consider also IL-frd)
+MI-hth - Promotion of misinformation that is likely to cause serious bodily harm or death
+nudity | NS - Nudity & Sex
+NS-nud - Casual nudity
+NS-ero - Erotica
+NS-sex - Sex
+PN - Pornography
+PN-het - Heterosexual porn
+PN-gay - Gay male porn
+PN-les - Lesbian porn
+PN-bis - Bisexual porn
+PN-trn - Transexual porn
+PN-fnb - Gender-fluid / non-binary porn
+spam | SP - Spam
+SP-mod - Moderation report spam
+VI - Violence (actual or advocated)
+VI-hum - Violence towards a human being
+VI-ani - Violence towards a sentient animal
+
+
+
+Contexts:
+
+Contexts are situations where otherwise objectionable content may be allowed.
+
```

Comment:
**_Format:_ AB**
- where 'AB' is a mandatory 2 character, uppercase code of a major content type category